### ISSUE TITLE
Calculate matrix for projection in simulation

### ISSUE THREAD

**Issue by ddiez** [MarioniLab/MNN2017#13]
In the following line in simulateBatches.R:

https://github.com/MarioniLab/MNN2017/blob/08bc2f5fc196be197be47cb56b3e85eedae87acf/Simulations/simulateBatches.R#L40

should not that line be instead:

```
proj <- matrix(rnorm(ngenes*2), nrow=ngenes, ncol=2)
```

That is, although `ngenes * ncells` values are obtained with `rnorm()` only `ngenes * 2` are really used in the `proj` matrix.

**Comment by LTLA** [comment]
Yes, you're right, but it doesn't matter as `matrix` will just truncate when filling itself. The corresponding code in https://github.com/MarioniLab/FurtherMNN2018 should be more correct.
